resources/htmlTemplate/TocHTMLTemplate.html

```diff
@@ -134,20 +134,19 @@ <h1><a href="/" title="Home" rel="home">Software Package Data Exchange (SPDX)</a
 
       <h1>SPDX License List</h1>
 
-      <p>The SPDX License List is a list of commonly found licenses and exceptions used in free and open source and other collaborative software. The purpose of the SPDX License List is to enable 
-      easy and efficient identification of such licenses and exceptions in an SPDX document 
-      (or elsewhere). The SPDX License List includes a standardized short identifier, full name 
-      for each license, vetted license text, other basic information, and a canonical permanent 
-      URL for each license and exception. By providing a short identifier, users can efficiently 
-      refer to a license without having to redundantly reproduce the full license.  
+      <p>The SPDX License List is a list of commonly found licenses and exceptions used in free and open source and other collaborative software or documentation. The purpose of the SPDX License List is to enable 
+      easy and efficient identification of such licenses and exceptions in an SPDX document, in source files 
+      or elsewhere. The SPDX License List includes a standardized short identifier, full name, vetted license 
+	text including matching guidelines markup as appropriate, other information, and a canonical permanent 
```

Comment:
“matching guidelines markup” → “matching markup”?  Or “matching-guidelines markup”?

And this is a long enough list that I'd rather drop “other information”.  If information isn't important enough for an explicit listing here, I'm fine leaving it to folks reading the source to discover that low-importance information.

I'd also consider dropping “permanent”, since we change/add canonical URLs as the old ones go stale (although still no official policy on that, spdx/license-list-XML#433).